components/src/md3/MobileStepper/MobileStepper.tsx

```diff
@@ -0,0 +1,193 @@
+import React from 'react';
+import { View, StyleSheet, StyleProp, ViewStyle, ViewProps } from 'react-native';
+import { MD3Theme, ProgressBar, useTheme } from 'react-native-paper';
+import * as Colors from '@brightlayer-ui/colors';
+import { $DeepPartial } from '@callstack/react-theme-provider';
+import { Body1 } from '../../core/typography';
+import { getPrimary500 } from '../../core/utility/shared';
+
+const makeStyles = (
+    props: MobileStepperProps,
+    theme: MD3Theme
+): StyleSheet.NamedStyles<{
+    root: ViewStyle;
+    stepperContainer: ViewStyle;
+    circle: ViewStyle;
+    filled: ViewStyle;
+    progressBar: ViewStyle;
+    text: ViewStyle;
+}> =>
+    StyleSheet.create({
+        root: {
+            alignItems: 'center',
+            justifyContent: 'center',
+            flexDirection: 'row',
+            margin: 16,
+        },
+        stepperContainer: {
+            flexDirection: 'row',
+            flex: 1,
+            alignItems: 'center',
+            justifyContent: 'center',
+        },
+        circle: {
+            height: 8,
+            width: 8,
+            borderRadius: 8,
+            marginHorizontal: 4,
+            overflow: 'hidden',
+            backgroundColor: props.inactiveColor || (theme.dark ? theme.colors.onBackground : Colors.gray[200]),
```

Comment:
Use the custom "disabled" color for the background color

Reply:
There is no colour named "disabled", but onBackground colour is same colour as the disabled colour.

Reply:
(1)
> There is no colour named "disabled"

Yes, that is a custom-added color from the design side. If that variable does not exist in the theme, see if you can find a way to add that to the theme, or at least leave a comment here to update the color in the future.

(2) 
> onBackground colour is same colour as the disabled colour
This is what "onBackground" currently looks like on the design side. 

![image](https://github.com/etn-ccis/blui-react-native-component-library/assets/8997218/23449278-fef5-4461-a1a4-a03db8836a1b)

Reply:
Added custom disabled color variable with the Hex value mentioned above in MD3BluiLightTheme file. It's picking up the added color, but showing error as below so for that added a comment in the code.
<img width="778" alt="Screenshot 2023-10-12 at 4 24 47 PM" src="https://github.com/etn-ccis/blui-react-native-component-library/assets/130662346/f8a3e903-51dd-4f0d-bad4-156d6f692bd7">